static/app/views/releases/detail/overview/issues.tsx

```diff
@@ -49,22 +56,63 @@ type State = {
   issuesType: IssuesType;
   pageLinks?: string;
   onCursor?: () => void;
+  count: {
+    firstRelease: number;
+    release: number;
+    resolved: number;
+    unhandled: number;
+  };
 };
 
 class Issues extends Component<Props, State> {
-  state: State = {
-    issuesType: IssuesType.NEW,
-  };
+  state: State = this.getInitialState();
+
+  getInitialState() {
+    const {location} = this.props;
+    const query = location.query ? location.query.query : null;
+    const issuesTypeState = !query
+      ? IssuesType.NEW
+      : query.includes('first-release')
+      ? IssuesType.NEW
+      : query.includes('error.handled:0')
+      ? IssuesType.UNHANDLED
+      : query.includes('is:resolved')
+      ? IssuesType.RESOLVED
+      : IssuesType.ALL;
```

Comment:
Should we have in the URL not full queries, but just something like `?issueType=new`?